plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/dialogs/GotoObjectDialog.java

```diff
@@ -84,15 +87,39 @@ public GotoObjectDialog(Shell shell, DBCExecutionContext context, DBSObject cont
 
     @Override
     protected Control createExtendedContentArea(Composite parent) {
+        Composite composite = UIUtils.createComposite(parent, 1);
+        composite.setLayoutData(new GridData(GridData.FILL_BOTH));
+
+        createObjectTypesGroup(composite);
+
+        Button searchInCommentsCheckbox = UIUtils.createCheckbox(
+                composite,
+                UINavigatorMessages.dialog_search_objects_search_in_comments,
+                !doNotSearchInComments
+        );
```

Comment:
We use 4-space indentation for continuations.

You can configure code style for Intellij IDEA in <kbd>Editor</kbd> &rArr; <kbd>Code Style</kbd> &rArr; <kbd>Java</kbd> &rArr; <kbd>Continuation indent</kbd>.